### PR TITLE
Add karaf-rest-example with Jakarta Jersey client

### PR DESCRIPTION
**Pull request by benroe89**

### CODE REVIEW

**Comment by jbonofre** [comment]
retest this please

**Comment by fpapon** [comment]
@benroe89 you have to rebase

File: examples/karaf-rest-example/karaf-rest-example-client/karaf-rest-example-client-cxf/src/main/java/org/apache/karaf/examples/rest/client/cxf/AddBookingCommand.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.apache.karaf.examples.rest.client.cxf;
+package org.apache.karaf.examples.rest.client.jersey;
```
Comment (fpapon) [inline examples/karaf-rest-example/karaf-rest-example-client/karaf-rest-example-client-cxf/src/main/java/org/apache/karaf/examples/rest/client/cxf/AddBookingCommand.java:17]: package should be `org.apache.karaf.examples.rest.client.cxf`

File: examples/karaf-rest-example/karaf-rest-example-client/karaf-rest-example-client-cxf/src/main/java/org/apache/karaf/examples/rest/client/cxf/ListBookingCommand.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.apache.karaf.examples.rest.client.cxf;
+package org.apache.karaf.examples.rest.client.jersey;
```
Comment (fpapon) [inline examples/karaf-rest-example/karaf-rest-example-client/karaf-rest-example-client-cxf/src/main/java/org/apache/karaf/examples/rest/client/cxf/ListBookingCommand.java:17]: package should be `org.apache.karaf.examples.rest.client.cxf`

**Comment by benroe89** [comment]
So, should be rebased properly and the packages were renamed as suggested.

**Comment by jbonofre** [comment]
Replaced by #1209

**Pull request closed without merging**